pkg/reconciler/v1alpha1/clusteringress/clusteringress.go

```diff
@@ -113,10 +110,10 @@ func NewController(
 	})
 
 	c.Logger.Info("Setting up ConfigMap receivers")
-	resyncIngressesOnIstioConfigChange := configmap.TypeFilter(&config.Istio{})(func(string, interface{}) {
+	resyncIngressesOnConfigChange := configmap.TypeFilter(&config.Config{})(func(string, interface{}) {
```

Comment:
I don't think this is right, shouldn't it be the list of types?

Reply:
Fixed it.